.bash_prompt

```diff
@@ -164,7 +164,14 @@ sexy_bash_prompt_branch_exists () {
 sexy_bash_prompt_parse_git_ahead () {
   # Grab the local and remote branch
   branch="$(sexy_bash_prompt_get_git_branch)"
-  remote_branch="origin/$branch"
+  
+  # remote_branch="origin/$branch"
+  # Compute the remote tracking branch, if no remote branch then echo 0
+  remote_branch="$(git rev-parse --symbolic-full-name "@{upstream}" 2>/dev/null )"
```

Comment:
So I've added the fallback to origin but tests are still failing. Curiously all tests passed when for-each-ref was in use.